test/lib/mayaUsd/render/mayaToHydra/testNurbsPrimitives.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2024 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import maya.cmds as cmds
+import fixturesUtils
+import mtohUtils
+
+class TestNurbsPrimitives(mtohUtils.MtohTestCase):
+    # MayaHydraBaseTestCase.setUpClass requirement.
+    _file = __file__
+
+    IMAGE_DIFF_FAIL_THRESHOLD = 0.1
+    IMAGE_DIFF_FAIL_PERCENT = 0.75
+
+    def compareSnapshot(self, referenceFilename, cameraDistance=10):
+        self.setBasicCam(cameraDistance)
+        cmds.refresh()
+        self.assertSnapshotClose(referenceFilename, self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)
+
+    def setupScene(self, nurbsCreationCallable):
+        cmds.loadPlugin('ArubaTessellator') # Loaded by default in Maya, changes NURBS vertices calculations
+        self.setHdStormRenderer()
+        nurbsResult = nurbsCreationCallable()
+        cmds.refresh()
+        return nurbsResult
```

Comment:
Nit-pick: looks like all the call sites are using the index 1 value from the return list, may as well save some typing and return nurbsResult[1].

Reply:
Yeah I originally had it that way in case I needed the nurbs node, but in the end I didn't. Refactored it to directly return the makeNurb node.